meldekortdomene/src/main/kotlin/no/nav/aap/varsel/VarselService.kt

```diff
@@ -0,0 +1,168 @@
+package no.nav.aap.varsel
+
+import no.nav.aap.kelvin.KelvinSakRepository
+import no.nav.aap.kelvin.KelvinSakService
+import no.nav.aap.komponenter.repository.RepositoryProvider
+import no.nav.aap.lookup.gateway.GatewayProvider
+import no.nav.aap.meldeperiode.Meldeperiode
+import no.nav.aap.sak.FagsakReferanse
+import no.nav.aap.sak.Fagsaknummer
+import no.nav.aap.sak.FagsystemNavn
+import no.nav.aap.utfylling.Utfylling
+import org.slf4j.LoggerFactory
+import java.time.Clock
+import java.time.Instant
+import no.nav.aap.komponenter.config.requiredConfigForKey
+import no.nav.aap.utfylling.UtfyllingRepository
+import java.time.LocalDate
+import java.time.LocalTime
+import java.time.ZoneId
+import java.util.*
+
+class VarselService(
+    private val kelvinSakService: KelvinSakService,
+    private val kelvinSakRepository: KelvinSakRepository,
+    private val varselRepository: VarselRepository,
+    private val utfyllingRepository: UtfyllingRepository,
+    private val varselGateway: VarselGateway,
+    private val clock: Clock
+) {
+    constructor(repositoryProvider: RepositoryProvider, gatewayProvider: GatewayProvider, clock: Clock) : this(
+        kelvinSakService = KelvinSakService(repositoryProvider, gatewayProvider, clock),
+        kelvinSakRepository = repositoryProvider.provide(),
+        varselRepository = repositoryProvider.provide(),
+        utfyllingRepository = repositoryProvider.provide(),
+        varselGateway = gatewayProvider.provide(),
+        clock = clock
+    )
+
+    private val log = LoggerFactory.getLogger(javaClass)
+
+    fun planleggFremtidigeVarsler(saksnummer: Fagsaknummer) {
+        val meldeplikt = kelvinSakRepository.hentMeldeplikt(saksnummer)
+
+        val meldeperioder = kelvinSakService.hentMeldeperioder(FagsakReferanse(FagsystemNavn.KELVIN, saksnummer))
+            .filter { meldeplikt.contains(it.meldevindu) }
+
+        if (meldeperioder.size != meldeplikt.size) {
+            log.warn("Meldevindu i meldeperioder samsvarer ikke med meldeplikt-perioder fra Kelvin")
```

Comment:
```suggestion
            log.warn("Meldevindu i meldeperioder samsvarer ikke med meldeplikt-perioder fra Kelvin. Saksnummer: $saksnummer")
```

Gjerne ha med kontekst i loggmeldinger